testsuite/objects/__init__.py

```diff
@@ -35,40 +67,45 @@ class Rule:
     value: str
 
 
-class Value:
-    """Dataclass for specifying a Value in Authorization, can be either constant or value from AuthJson (jsonPath)"""
+@dataclass
+class ValueBase:
+    """
+    Abstract Dataclass for specifying a Value in Authorization,
+    can be either static or reference to value in AuthJson.
+    """
 
-    # pylint: disable=invalid-name
-    def __init__(self, value=None, jsonPath=None) -> None:
-        super().__init__()
-        if not (value is None) ^ (jsonPath is None):
-            raise AttributeError("Exactly one of the `value` and `jsonPath` argument must be specified")
-        self.value = value
-        self.jsonPath = jsonPath
+
+@dataclass
+class ValueStatic(ValueBase):
+    """Dataclass for static Value. Can be anything, eg. string, integer, list, dict..."""
+
+    value: Any
```

Comment:
`Any` type seems too broad. I'd rather `Union`'d all possible types.

Reply:
I see the point. Anyway `Union` would be a) HUUUUGE b) something would be forgotten (I bet on `bytes`)

Reply:
I see the point. Anyway `Union` would be a) HUUUUGE b) something would be forgotten (I bet on `bytes`)

Reply:
I am ok with `Any`, but if you want to have all types in Union I would prefer custom type called `JSONValues`, which would contain the Union and then only reference this new type in Value

Reply:
I created new type Union